README.md

```diff
@@ -1,4 +1,4 @@
-# fluent-plugin-logdna
+# fluent-plugin-logdna Gem Version
```

Comment:
Is there a non-gem version of fluent plugins?

Reply:
Oh I see - windows

Reply:
Sorry, I'm still a bit confused what `Gem Version` is trying to indicate. Fluentd plugins are gems.
Maybe this is just a organization problem.

I think the top level h1 title should just be the name of the project `fluent-plugin-logdna`

then break down installation instructions

```txl
h2. Installation

h3. Using RubyGems

h3. Using td-agent (windows)

h4. Using FPM
```

Something like that maybe? it is a little clunky current